drf_braces/fields/custom.py

```diff
@@ -73,15 +73,18 @@ class RoundedDecimalField(fields.DecimalField):
     to two decimal places.
     """
 
-    def __init__(self, max_digits=100, decimal_places=2, *args, **kwargs):
-        super(fields.DecimalField, self).__init__(
+    def __init__(self, max_digits=None, decimal_places=2, *args, **kwargs):
+        super(RoundedDecimalField, self).__init__(
             max_digits=max_digits,
             decimal_places=decimal_places,
             *args, **kwargs
         )
 
-    def to_internal_value(self, data):
-        return self.quantize(super(RoundedDecimalField, self).to_internal_value(data))
+    def quantize(self, value):
+        if self.max_digits is None and isinstance(value, float):
```

Comment:
this is no longer a `DecimalField` if it will return `float` in some cases. it must return `float` in all cases unless its not required field and value is not provided